src/content/cloud.mdx

```diff
@@ -1,28 +0,0 @@
----
-title: Okteto Cloud
-description: Okteto Cloud gives you free access to secure Kubernetes namespaces, fully integrated with remote development capabilities
-sidebar_label: Okteto Cloud
-id: cloud
----
-
-[Okteto Cloud](https://cloud.okteto.com/#/?origin=docs) gives you free access to secure Kubernetes namespaces, fully integrated with remote development capabilities.
-Develop your applications in Okteto Cloud and forget about slow and tedious local development forever.
-
-With Okteto Cloud you can:
```

Comment:
My only concern is not having this list of features anywhere. Do you think they're essential? 

If yes, maybe we can have them on the Dev Environments Overview page since most of them are relevant for that?

Reply:
I don't think they are that relevant, since the features are described on the docs. But maybe a quick summary?